libs/SalesforceSDKCore/Configuration/SalesforceSDKCore-Common-Static.xcconfig

```diff
@@ -11,4 +11,4 @@
 INSTALL_PATH = /
 SKIP_INSTALL = YES
 
-HEADER_SEARCH_PATHS = $(BUILT_PRODUCTS_DIR)/Headers $(TARGET_BUILD_DIR)/Headers $(inherited)
+// HEADER_SEARCH_PATHS = $(BUILT_PRODUCTS_DIR)/Headers $(TARGET_BUILD_DIR)/Headers $(inherited)
```

Comment:
I see. @khawkins has been doing some work on cleaning up our targets and fixing everything in the workspace, so I'll defer to him on this.